src/gtimelog/tests/components/test_utils.py

```diff
@@ -0,0 +1,94 @@
+import doctest
+import unittest
+
+# from gtimelog.main import require_version
+# require_version('Gtk', '3.0')
+# require_version('Soup', '2.4')
+# require_version('Secret', '1')
+# from gi.repository import GLib
```

Comment:
Importing GUI stuff in unit tests was problematic in the past for Debian package builds.

Maybe something could be done with build-depending on xvfb-run.  I'd like to hear the Debian package maintainer's opinion on this.

Cc: @andrewshadura

Reply:
I removed the test btw. This must be done later on